src/js/utils/mixins.js

```diff
@@ -1,7 +1,11 @@
 import { css } from 'styled-components';
 
-export const parseMetricToNum = fontAsString =>
-  parseFloat(fontAsString.replace(/[^0-9/.]/g, ''), 10);
+export const parseMetricToNum = fontAsString => {
+  if (fontAsString.match(/\s/)) {
+    console.warn('This feature takes only 1 value and should have no spaces');
```

Comment:
another thing i thought today, the message is not clear. The user won't be able to know what happened. maybe we can give some more context (This feature = ?) and even mention the value we tried to parse.

On a side note i also think we should warn only in development mode so we don't pollute user console in production. What do you think?

Reply:
I like the idea of keeping warning to `development`only for now. About the message I made it more abstract in language & including argument:
https://github.com/crashuniverse/grommet/blob/fix/mixins-parseMetricToNum/src/js/utils/mixins.js#L4:L7